WalletWasabi/Transactions/OperationMerger.cs

```diff
@@ -0,0 +1,63 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Transactions
+{
+	public static class OperationMerger
+	{
+		/// <summary>
+		/// Merges the operations in a way that it leaves the order, but two consequtive remove or append cannot follow each other.
```

Comment:
```suggestion
		/// Merges the operations in a way that it leaves the order, but two consecutive remove or append cannot follow each other.
```

Reply:
Actually I will do a PR with just the txbuilder soon. I only added it to this PR to be able to test better.

Reply:
Here: https://github.com/zkSNACKs/WalletWasabi/pull/2401